src/app/ApplicationTemplate.Droid/MainActivity.cs

```diff
@@ -24,12 +24,6 @@ public class MainActivity : ApplicationActivity
 		protected override void OnCreate(Bundle bundle)
 		{
 			base.OnCreate(bundle);
-
-#if (IncludeFirebaseAnalytics)
```

Comment:
Why are you removing this comment?

Reply:
I think comment is there since official documentation will say to call this in OnCreate, but we want to have a comment there to explain why we're not calling it here.

Reply:
The reason I removed the comment at first is because the code may not even exists with the new sdk (no fabric). I will double check if it is still relevant with the new sdk

Reply:
Well in the most recent documentation it does not say to call anything on the OnCreate method by default [start android doc](https://firebase.google.com/docs/crashlytics/get-started?platform=android.) The only thing is that you can control whether manually when to report which we have to manually set [manual report doc](https://firebase.google.com/docs/crashlytics/customize-crash-reports?platform=android#enable-reporting) which I don't think it's necessary for now. There's also the documentation for the FirebaseCrashlytics which replace pretty much the Crashlytics entity [FirebaseCrashlytics docs](https://firebase.google.com/docs/reference/android/com/google/firebase/crashlytics/FirebaseCrashlytics)